src/gui/mzroll/pollyelmaveninterface.cpp

```diff
@@ -210,29 +215,28 @@ void PollyElmavenInterfaceDialog::call_initial_EPI_form() {
     QCoreApplication::processEvents();
 }
 
-void PollyElmavenInterfaceDialog::handleAuthentication(QString status) {
+void PollyElmavenInterfaceDialog::handleAuthentication(QString status)
+{
     if (status == "ok") {
         _loadingDialog->statusLabel->setText("Fetching your projects..");
         QCoreApplication::processEvents();
-    }
-    else {
-            _loadingDialog->statusLabel->setText("Authentication failed. Please login again.");
-            QCoreApplication::processEvents();
-            logout();
+    } else {
+        _loadingDialog->statusLabel->setText("Authentication failed. Please login again.");
+        QCoreApplication::processEvents();
+        logout();
     }
 }
 
-void PollyElmavenInterfaceDialog::handleResults(QVariantMap projectnamesIdMap){
+void PollyElmavenInterfaceDialog::handleResults(QVariantMap projectnamesIdMap)
+{
     projectNamesId = projectnamesIdMap;
```

Comment:
@shubhra-agrawal 
In case of No internet connection, projectNamesId will be an empty QMap().
We should check for it in later steps, I mean in the startupDataLoad function.
Otherwise EPI will not be able to detect internet connection issues which is what is happening right now, as reported by @rish9511 and @saifulbkhan .

Reply:
@Ky5595 Empty QMap of project list is not a good indicator of no internet connection. The authentication function should check for this and return an error.